### ISSUE TITLE
Can't install Minikube 

### ISSUE THREAD

**Issue by lalib** [kubernetes/minikube#2132]
**Environment**:

**Minikube version** (use `minikube version`): v0.23.0
 **OS** (e.g. from /etc/os-release): macOS 10.12.6

**What happened**:
When i try to install minikube, i get an exception which can be seen in the logs section.

**What you expected to happen**:
To install minikube.

**How to reproduce it** (as minimally and precisely as possible):
When i run `brew cask install minikube`

**Output of `minikube logs` (if applicable)**:
```
==> Satisfying dependencies
All Formula dependencies satisfied.
==> Downloading https://storage.googleapis.com/minikube/releases/v0.23.0/minikube-darwin-amd64
Already downloaded: /Users/balp/Library/Caches/Homebrew/Cask/minikube--0.23.0
==> Verifying checksum for Cask minikube
==> Installing Cask minikube
==> Linking Binary 'minikube-darwin-amd64' to '/usr/local/bin/minikube'.
Error: Command failed to execute!

==> Failed command:
/usr/bin/xattr -w com.apple.metadata:kMDItemAlternateNames ("minikube") /usr/local/Caskroom/minikube/0.23.0/minikube-darwin-amd64

==> Standard Output of failed command:


==> Standard Error of failed command:
Traceback (most recent call last):
  File "/usr/bin/xattr-2.7", line 7, in <module>
    from pkg_resources import load_entry_point
ImportError: No module named pkg_resources


==> Exit status of failed command:
#<Process::Status: pid 17162 exit 1>
```

**Comment by andrei-dascalu** [comment]
Maybe similar to what I experienced? https://github.com/kubernetes/minikube/issues/2120

**Comment by terryfernz** [comment]
I tried installing the previous version of minikube 0.22.2 which after running a minikube start --vm-driver=vmwarefusion automatically started to upgrade to the latest version 0.23 and with that minikube seems to be now up and running.

Terences-Mac-Pro:~ Terry$ minikube start --vm-driver=vmwarefusion
There is a newer version of minikube available (v0.23.0).  Download it here:
https://github.com/kubernetes/minikube/releases/tag/v0.23.0

To disable this notification, run the following:
minikube config set WantUpdateNotification false
Starting local Kubernetes v1.7.5 cluster...
Starting VM...
Downloading Minikube ISO
 70.67 MB / 139.09 MB [======================>---------------------]  50.81% 40s 139.09 MB / 139.09 MB [============================================] 100.00% 0s
Getting VM IP address...
Moving files into cluster...
Setting up certs...
Connecting to cluster...
Setting up kubeconfig...
Starting cluster components...
Kubectl is now configured to use the cluster.
Terences-Mac-Pro:~ Terry$  minikube status
minikube: Running
cluster: Running
kubectl: Correctly Configured: pointing to minikube-vm at xxx.xx.xxx.xxx

**Comment by lalib** [comment]
@terryfernz , @andrei-dascalu it is the same. I have tried your solutions and did not work for me.
